scrapers/src/crime/kc/data/kc-2017-crime.csv

```diff
@@ -0,0 +1,7308 @@
+year,month,day,description,count
+2017,1,1,Agg Assault - Domest,4
+2017,1,1,Aggravated Assault (,8
+2017,1,1,Armed Robbery,3
+2017,1,1,Arson,3
+2017,1,1,Attempt Suicide by C,1
+2017,1,1,Attempt Suicide by O,1
+2017,1,1,Attempt Suicide by S,1
+2017,1,1,Auto Theft,11
+2017,1,1,Auto Theft Outside S,2
+2017,1,1,Burglary - Non Resid,1
+2017,1,1,Burglary - Residence,4
+2017,1,1,Credit/Debit/ATM Car,1
+2017,1,1,Disorderly Conduct,6
+2017,1,1,Fraud/Confidence Gam,1
+2017,1,1,Hit and Run of a Per,1
+2017,1,1,Intimidation,2
+2017,1,1,Misc Violation,3
+2017,1,1,Non Agg Assault Dome,22
+2017,1,1,Non Aggravated Assau,12
+2017,1,1,Possession of Drug E,2
+2017,1,1,Possession/Sale/Dist,3
+2017,1,1,Property Damage,28
+2017,1,1,Resisting an Officer,3
+2017,1,1,Sex Off Fondle - mol,1
+2017,1,1,Statutory Rape,1
+2017,1,1,Stealing All Other,5
+2017,1,1,Stealing Auto Parts/,3
+2017,1,1,Stealing From Auto,7
+2017,1,1,Stealing Shoplifting,5
+2017,1,1,Stealing from Buildi,4

```

Comment:
Looks like a bunch of the descriptions are cut off. Is that coming from our end or the data source?